<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>631</width>
      <height>521</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>451</width>
       <height>122</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelNombre">
        <property name="text">
         <string>Nombre de la tienda: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelDireccionWeb">
        <property name="text">
         <string>Dirección Web:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelDireccionFisica">
        <property name="text">
         <string>Dirección física:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelTelefono">
        <property name="text">
         <string>Teléfono:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtNombreTienda"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtDireccionWeb"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txtDireccionFisica"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="txtTelefono"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btnAgregar">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>400</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Agregar producto</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConsultar">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>400</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Consultar producto</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnModificar">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>400</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Modificar producto</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEliminar">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>400</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Eliminar producto</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAbrirArchivo">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>460</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Abrir archivo binario</string>
     </property>
    </widget>
    <widget class="QListWidget" name="lista">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>190</y>
       <width>451</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelInformativo">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>170</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Lista de productos:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnGuardarArchivo">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>460</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Guardar lista en Archivo</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>624</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
